bcda/cli.go

```diff
@@ -349,10 +354,13 @@ func importCCLFDirectory(filePath string) (success, failure, skipped int, err er
 				failure++
 			} else {
 				log.Infof(fmt.Sprintf("Successfully imported CCLF9 file: %v", cclf9))
+				cclf9.imported = true
 				success++
 			}
 		}
 	}
```

Comment:
in reference to the CCLF0 comment above, we could mark CCLF0 imported = true here if CCLF8 AND CCLF9 are imported

cclf0.imported = cclf8.imported && cclf9.imported

Reply:
exactly